galley/pkg/config/analysis/analyzers/testdata/injection.yaml

```diff
@@ -44,3 +44,14 @@ spec:
   containers:
   - image: gcr.io/google-samples/microservices-demo/adservice:v0.1.1
     name: server
+---
+# Pod that explicitly disables injection. Should not generate a warning.
+kind: Pod
+metadata:
+  name: podinjectiondisabled
+  namespace: default
+  sidecar.istio.io/inject: "false"
```

Comment:
This isn't right, it should be under `annotations:` I think?

Reply:
You need `istioctl analyze` and it would tell you that :slightly_smiling_face:

Reply:
Hah, whoops! Yeah I thought I'd run the tests successfully before I pushed but I suspect I actually ran the wrong ones. Fixed now.